jenkins/Jenkinsfile.nightly

```
@@ -52,7 +52,7 @@ pipeline {
 			when {
 				anyOf {
 					branch 'development'
-					branch 'feature/blockchain-agnostic'
+					branch 'feature/1157-prepare-lisk-service-v070-for-beta'
```

Is it possible to use a wildcard to run the jobs against any feature branch? Something like below:
Please check and apply the same below as well.
```suggestion
					branch 'feature/**'
```

---

I will check if possible and update the file.